type-o-matic/type-o-matic.csproj

```diff
@@ -1,151 +1,127 @@
+    </Compile>
+    <Compile Include="..\SwiftReflector\Demangling\MatchRule.cs">
+      <Link>Demangling\MatchRule.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\Demangling\RuleRunner.cs">
+      <Link>Demangling\RuleRunner.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\Demangling\ContextAttribute.cs">
+      <Link>Demangling\ContextAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\Demangling\Enums.cs">
+      <Link>Demangling\Enums.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\StringSlice.cs">
+      <Link>SwiftType\StringSlice.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\SwiftClassName.cs">
+      <Link>SwiftType\SwiftClassName.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\SwiftType.cs">
+      <Link>SwiftType\SwiftType.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\SwiftName.cs">
+      <Link>SwiftType\SwiftName.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\ExceptionTools\ErrorHelper.cs">
+      <Link>SwiftType\ErrorHelper.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\Extensions.cs">
+      <Link>SwiftType\Extensions.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\MachOHawley.cs">
+      <Link>MachOHawley.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\MachOHelpers.cs">
+      <Link>MachOHelpers.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\ReflectorError.cs">
+      <Link>SwiftType\ReflectorError.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\PunyCode.cs">
+      <Link>SwiftType\PunyCode.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\Importing\TypeAggregator.MacOS.cs">
+      <Link>TypeAggregator.MacOS.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\Demangling\Swift5Demangler.cs">
+      <Link>Demangling\Swift5Demangler.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\Demangling\Swift5NodeToTLDefinition.cs">
+      <Link>Demangling\Swift5NodeToTLDefinition.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\ExceptionTools\RuntimeException.cs">
+      <Link>SwiftType\RuntimeException.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftReflector\CompilationTarget.cs">
+      <Link>CompilationTarget.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Dynamo\Dynamo.csproj" />
+    <ProjectReference Include="..\SwiftRuntimeLibrary\SwiftRuntimeLibrary.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
+    <Folder Include="Demangling\" />
+    <Folder Include="SwiftType\" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Mono.Options" Version="6.12.0.148" />
+    <PackageReference Include="Mono.Cecil" Version="0.11.5" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="TypeMetadataSet.cs" />
```

Comment:
Why is this file being removed? Add a comment, or if it is no longer used entirely, delete the file.